### ISSUE TITLE
Indenting automated list bullets

### ISSUE THREAD

**Issue by eseiver** [ttscoff/nv#284]
Really like the automated list continuation feature. However nvALT acts inconsistently when indenting an automatically bulleted line. If I'm on the last line of the entire document in a bulleted list, hitting 'enter' then 'tab' will simply move the cursor further away from the newly generated asterisk. However, if it's not the last line, 'enter' then 'tab' will indent the bullet as well. Could you fix this so that 'enter' + 'tab' always indents an automated bullet? Many thanks!

**Comment by LostKobrakai** [comment]
:+1: for this. This makes automatic listing basicly useless for me.

**Comment by ttscoff** [comment]
Just use ⌘]

-Brett

**Comment by LostKobrakai** [comment]
That's an alternative, but not a very great one, depending on the keyboard layout. 
`CMD` + `Alt` + `5` vs `Tab`
`CMD` + `Alt` + `6` vs `Shift` + `Tab`
I'd rather take the latter variant.

**Comment by elasticthreads** [comment]
@LostKobrakai You're right that cmd-]  or cmd-[  are much more useful for US keyboard layouts.

But there are also menu items for Shifting the current paragraph to the right under the format menu. These will shift whatever line your cursor is on a tab stop to the left or right. Which is better than Tab because tab requires the cursor be at the beginning of the line to work.

AND:
You can change the KB shortcuts for these (if you don't use a US KB layout) in System Prefs-->Keyboard-->Shortcuts-->App Shortcuts
Create shortcuts for nvALT using "Shift Left" and "Shift Right"  as the Menu Title

**Comment by LostKobrakai** [comment]
I'll see. But to be clear: I've seen so many mac apps, that can use TAB in lists no matter of where the cursor is located. So it would for sure be quite nice.

**Comment by robertmunk** [comment]
I'd also like to see this fixed.  I notice that if i'm in the middle of a list and press enter to open up a new list item, _then_ press tab, the new list item I created is correctly indented.

I'm currently working around this by adding an empty list item at the end of my list so that Tab works correctly but this should really be fixed.

**Comment by nnnnicholas** [comment]
Hey @ttscoff, I love nvALT and I'm back using it after 8 years away in Windows and Linux.

Is there any chance we could get `tab to indent bulleted list` in a future update? I use Sublime and VSCode when I'm not writing in nvALT and they both handle bullets well (with Markdown plugins). Sublime in particular alternates between `-` `+` and `*` for the bullets at different levels of indentation, which `CMD + ]` does not do in nvALT. It's more natural for many text editing apps (Word, Pages, Notes, etc) to hit `return` for newline, then `tab` to indent, instead of `CMD + ]`. 

Thanks for considering it!
